srcpkgs/libgccjit/template

```diff
@@ -0,0 +1,255 @@
+# Template file for 'libgccjit'
+_majorver=10
+_minorver=${_majorver}.2
+_patchver=${_minorver}.1
+_gmp_version=6.2.0
+_mpfr_version=4.1.0
+_mpc_version=1.1.0
+_isl_version=0.21
+
+pkgname=libgccjit
+# we are using a 10-stable git snapshot alpine is using in order
+# to get regression fixes not yet incorporate into a stable release
+# it should be possible to switch back to stable with 10.3 or 11
+version=${_patchver}pre1
+revision=1
+wrksrc=gcc-${version/pre/_pre}
+short_desc="GNU Compiler Collection"
```

Comment:
That's not a good description

Reply:
is "Just-In-Time Compilation library for gcc" good?

Reply:
Yes

Reply:
Though GCC should be in caps.

Reply:
changed